### ISSUE TITLE
YouTube tracking removal not working?

### ISSUE THREAD

**Issue by SM91337** [newhouse/url-tracking-stripper#77]
Hi, I have noticed this extension is not working on YouTube right now, I am not sure if its because of something specific to my setup, I am running Opera with the extension that allows you to run Chrome extensions would that be a problem?

An important thing I should note, when I manually right click the url and select "Copy and Clean link" from the context menu it works fine (as it should) and the tracking gets removed.

So to sum up, I don't think automatic tracking removal is working on YouTube.

Example url: https://www.youtube.com/redirect?event=video_description&redir_token=pfsH9gZ730FoUkqi7XHqWNAi1zl8MTU0NTQwMzE4MEAxNTQ1MzE2Nzgw&q=https%3A%2F%2Fcbdfx.com%2Fproducts%2Fcbd-capsules-750mg&v=cB9SfzS56ms

Stripped url: https://cbdfx.com/products/cbd-capsules-750mg

Thanks!
